packages/object-map-keys/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Definitions by: Aryaman1706 <https://github.com/Aryaman1706>
+declare function map<T extends object = object>(
+  obj: T,
+  predicate: (value: any, key: string | number, object: T) => string | number
+): object;
+export = map;
```

Comment:
Recommend making this `export default map`. There was an issue sometime ago where it had to be converted to export default in lot of packages. @angus-c mind telling what it was 🤔?

Reply:
Yeah it was to make it sync with module package exports